src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,193 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchParams } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const getCenturiesParams = (sortName: string | null): SearchParams => {
+    const arrParams: string[] | null
+    = [...searchParams.getAll('centuries')];
+
+    if (!sortName) {
+      return { centuries: null };
+    }
+
+    if (!arrParams.includes(sortName)) {
+      arrParams.push(sortName);
+
+      return { centuries: arrParams };
+    }
+
+    arrParams.splice(arrParams.indexOf(sortName), 1);
```

Comment:
why do you need it?

Reply:
When all the buttons of the centuries are checked, this part of the code helps to get the correct array of values from the active buttons after deselecting any of them